lgc/state/PipelineState.cpp

```diff
@@ -1075,6 +1075,10 @@ unsigned PipelineState::getShaderWaveSize(ShaderStage stage) {
       waveSize = 64;
     }
 
+    if (getTargetInfo().getGfxIpVersion().minor >= 3 && stage == ShaderStageCompute) {
```

Comment:
{} is not needed. Better to add some comments to explain why we do this, such as:
// According to experimental data from performance tuning, wave64 is more efficient than wave32 in most cases.
// Hence, we set the wave size to wave64 by default.

Reply:
Sure!